src/components/ManagerPage/AppsList.js

```diff
@@ -264,9 +310,12 @@ class AppsList extends PureComponent<Props, State> {
       </Box>
     ) : status === 'success' ? (
       <Box grow align="center" justify="center" mt={5}>
-        <Box color="positiveGreen">
-          <CheckCircle size={44} />
-        </Box>
+        <IconWrapper>
+          <AppCheck>
+            <CheckFull size={24} />
+          </AppCheck>
+          <AppIcon src={this.getIconUrl()} />
```

Comment:
live-common already exposes a method for that, please use it:
https://github.com/LedgerHQ/ledger-live-common/blob/master/src/manager.js#L32